maintainers/maintainer-list.nix

```diff
@@ -5309,6 +5309,11 @@
     githubId = 848000;
     name = "éclairevoyant";
   };
+  ecklf = {
+    github = "ecklf";
+    githubId = 8146736;
+    name = "Florentin Eckl";
+  };
```

Comment:
Please provide another means of communication, e.g. `email`.
See https://github.com/NixOS/rfcs/pull/167.